content/docs/intro/concepts/programming-model.md

```diff
@@ -1,4555 +1,7 @@
 ---
 title: "Programming Model"
 meta_desc: An in depth overview of the the Pulumi Programming Model and common terms associated with the platform.
-menu:
-  intro:
-    parent: concepts
-    weight: 1
 
 aliases: ["/docs/reference/programming-model/"]
```

Comment:
I _think_ we want to add a top-level frontmatter param of `redirect_to: /docs/intro/concepts` here, right? In order to have links to this page end up on the new page? That would ensure that when we deploy this to production, requests for `/docs/intro/concepts/programming-model` will result in proper 301s to the new location.

Reply:
Yeah, I made that change in the follow-up commit 34197c9abed16f5c7b7af0d05fb389fb851e6e28. I tested it out locally and it seemed to work, but let me know if we need to modify it.